include/net/net_event.h

```diff
@@ -142,6 +142,25 @@ enum net_event_ipv4_cmd {
 #define NET_EVENT_IPV4_ROUTER_ADD				\
 	(_NET_EVENT_IPV4_BASE |	NET_EVENT_IPV4_CMD_ROUTER_ADD)
 
+/* L4 network events */
+#define _NET_L4_LAYER		NET_MGMT_LAYER_L4
+#define _NET_L4_CORE_CODE	0x114
+#define _NET_EVENT_L4_BASE	(NET_MGMT_EVENT_BIT |			\
```

Comment:
Nitpick: In order not to get another round of renames, names with leading '_' should not be used.

Reply:
I suggest that this PR is merged with the underscores, and then a later PR will remove all the underscores in this file (there are many of them).